common/src/main/java/com/microsoft/identity/common/adal/internal/tokensharing/TokenShareUtility.java

```diff
@@ -224,9 +222,9 @@ public void saveOrgIdFamilyRefreshToken(@NonNull final String ssoStateSerializer
                     }
                 });
 
-        final Pair<MicrosoftAccount, MicrosoftRefreshToken> resultPair = resultFuture.get();
+        final Map.Entry<MicrosoftAccount, MicrosoftRefreshToken> resultKeyValuePair = resultFuture.get();
```

Comment:
I guess we could just call it `result`

```suggestion
        final Map.Entry<MicrosoftAccount, MicrosoftRefreshToken> result = resultFuture.get();
```